backend/benefit/applications/models.py

```diff
@@ -881,3 +882,34 @@ class ReviewState(models.Model):
 class AhjoSetting(TimeStampedModel):
     name = models.CharField(max_length=255, unique=True)
     data = JSONField()
+
+
+class AhjoStatus(TimeStampedModel):
+    """
+    Ahjo status of the application
+    """
+
+    status = models.CharField(
+        max_length=64,
+        verbose_name=_("status"),
+        choices=AhjoStatus.choices,
+        default=AhjoStatus.SUBMITTED_BUT_NOT_SENT_TO_AHJO,
+        db_index=True,
```

Comment:
I think indexing choices field isn't very beneficial.

Reply:
Ok, I was thinking of a very hypthetical scenario where thousands of applications need to be queried by their latest ahjo status. But it's probably  not very realistic.